sql/mob_skills.sql

```diff
@@ -2156,7 +2156,7 @@ INSERT INTO `mob_skills` VALUES (2132,434,'replicator',0,7.0,2000,0,16,4,0,0,0,0
 -- INSERT INTO `mob_skills` VALUES (2140,1884,'peacebreaker',0,7.0,2000,1500,4,0,0,0,0,0,0);
 INSERT INTO `mob_skills` VALUES (2141,1465,'radiant_sacrament',0,20.0,2000,1000,4,4,0,0,0,0,0);
 INSERT INTO `mob_skills` VALUES (2142,1471,'mega_holy',1,18.0,2000,1500,4,0,0,0,0,0,0);
-INSERT INTO `mob_skills` VALUES (2143,1467,'perfect_defense',1,10.0,2000,1000,1,0,30,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (2143,152,'perfect_defense',1,18.0,2000,1000,1,0,30,0,0,0,0);
```

Comment:
Definitely don't want to change this animationID, as this is the mobskill that is very likely from captures

Reply:
Reverted the mob skill back.